src/resources/Search/Search.ts

```diff
@@ -10,6 +16,15 @@ export default class Search extends Ressource {
         return this.api.post<TokenModel>(`${Search.baseUrl}/token?organizationId=${API.orgPlaceholder}`, tokenParams);
     }
 
+    listFields(params?: SearchListFieldsParams) {
+        return this.api.get<SearchListFieldsResponse>(
+            this.buildPath(`${Search.baseUrl}/fields`, {
+                ...params,
+                organizationId: params?.organizationId ?? this.api.organizationId,
```

Comment:
Should the use case where `params.organizationId == null` be covered in the unit tests?

Reply:
You're right, I'll add a new test